chart/templates/back/deployment.yaml

```diff
@@ -0,0 +1,171 @@
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  key: {{ .Values.global.postgres.kyoo_back.userKey }}
+                  name: {{ .Values.global.postgres.kyoo_back.existingSecret }}
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: {{ .Values.global.postgres.kyoo_back.passwordKey }}
+                  name: {{ .Values.global.postgres.kyoo_back.existingSecret }}
+            - name: POSTGRES_DB
+              value: {{ .Values.global.postgres.database }}
+            - name: POSTGRES_SERVER
+              value: {{ .Values.global.postgres.host }}
+            - name: POSTGRES_PORT
+              value: "{{ .Values.global.postgres.port }}"
+            - name: RABBITMQ_DEFAULT_USER
+              valueFrom:
+                secretKeyRef:
+                  key: {{ .Values.global.rabbitmq.kyoo_back.userKey }}
+                  name: {{ .Values.global.rabbitmq.kyoo_back.existingSecret }}
+            - name: RABBITMQ_DEFAULT_PASS
+              valueFrom:
+                secretKeyRef:
+                  key: {{ .Values.global.rabbitmq.kyoo_back.passwordKey }}
+                  name: {{ .Values.global.rabbitmq.kyoo_back.existingSecret }}
+            - name: RABBITMQ_HOST
+              value: {{ .Values.global.rabbitmq.host }}
+            - name: RABBITMQ_PORT
+              value: "{{ .Values.global.rabbitmq.port }}"
+            - name: MEILI_HOST
+              value: "{{ .Values.global.meilisearch.proto }}://{{ .Values.global.meilisearch.host }}:{{ .Values.global.meilisearch.port }}"
+            - name: MEILI_MASTER_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: {{ .Values.global.meilisearch.kyoo_back.masterkeyKey }}
+                  name: {{ .Values.global.meilisearch.kyoo_back.existingSecret }}
+            {{- with (concat .Values.global.extraEnv .Values.back.kyoo_back.extraEnv) }}
+              {{- toYaml . | nindent 12 }}
+            {{- end }}
+          ports:
+            - name: main
+              containerPort: 5000
+              protocol: TCP
+          {{- with .Values.back.kyoo_back.livenessProbe }}
+          livenessProbe:
+            {{- toYaml .Values.back.kyoo_back.livenessProbe | nindent 12 }}
+          {{- end }}
+          {{- with .Values.back.kyoo_back.readinessProbe }}
+          readinessProbe:
+            {{- toYaml .Values.back.kyoo_back.readinessProbe | nindent 12 }}
+          {{- end }}
+          {{- with .Values.back.kyoo_back.resources }}
+          resources:
+            {{- toYaml .Values.back.kyoo_back.resources | nindent 12 }}
+          {{- end }}
+          {{- with .Values.back.kyoo_back.containerSecurityContext }}
+          securityContext:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            {{- with .Values.media.volumeMounts }}
```

Comment:
The back does not need the media volume

Reply:
Will remove tomorrow and also update the DIAGRAMS.md (probably in a separate PR)

Reply:
Removed backend mount and also updated diagram